lib/discoverable.coffee

```diff
@@ -153,7 +153,10 @@ isLoopbackInterface = (address) ->
 	foundIntf = _.find os.networkInterfaces(), (intfInfo) ->
 		_.find intfInfo, address: address
 
-	if not foundIntf? or  not (validFamily = _.find(foundIntf, family: family)?.internal)? then return null else return validFamily
+	if not foundIntf? or  not (validFamily = _.find(foundIntf, family: family)?.internal)?
+		return null
+
+	validFamily
```

Comment:
Can you add a `return` here, I find on longer functions it is much clearer/more readable to be explicit